public/app/screens/main/sections/devices/device/device.jsx

```diff
@@ -28,10 +28,12 @@ var Device = React.createClass({
         control = SliderControl;
         break;
     }
-    return control ? React.createElement(control, {device: this.props.device}) : (<div>Unknown type</div>);
+    return control
+      ? React.createElement(control, { device: this.props.device })
+      : (<div>Unknown type</div>);
   },
```

Comment:
This however is a good line break. Is it however being enforce by the style? Wrapping lines when using the ternary operator should however left to the developer.